mantidproject/mantid
# Use of reference files in PEARL system test

**Description of work.**

This PR enable comparisons again reference files for the `PearlPowderDiffractionScriptTest` system test, comparison being carried out for focus as well as the calibration system test. Recommend to merge this PR before merging #16590 .

**To test:**

Code Check
Check that system test build pass
No further pylint violation generated
If possible run `PearlPowderDiffractionScriptTest` system test locally



Fixes #16459 
_Does not need to be in the release notes._



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
